buildSrc/subprojects/buildquality/src/main/kotlin/org/gradle/gradlebuild/buildquality/DependencyVulnerabilitiesPlugin.kt

```diff
@@ -27,7 +26,7 @@ import org.owasp.dependencycheck.gradle.extension.DependencyCheckExtension
 open class DependencyVulnerabilitiesPlugin : Plugin<Project> {
```

Comment:
Can we remove this completely? It is not used, out of date (selected plugin version of 'org.owasp.dependencycheck' does not work with current wrapper anymore) and not included in any CI pipeline.